sdk/eventhubs/azure-messaging-eventhubs/test/eventhubs-stress-test/deploy.ps1

```diff
@@ -0,0 +1,2 @@
+Set-Location $PSScriptRoot
+pwsh "../../../../../eng/common/scripts/stress-testing/deploy-stress-tests.ps1"
```

Comment:
I get this is work in progress.  Would be nice to use explicit path, i.e. {some_root_val}/eng/.....

Reply:
That's what azure core uses in its deploy.ps1 script as well. But you're right, it would be nice to have a "root" environment variable of some kind.